yaksh/templates/yaksh/courses.html

```diff
@@ -3,23 +3,69 @@
 {% block pagetitle %} Courses {% endblock pagetitle %}
 {% block script %}
 <script>
+    //Highlight Selected Option in Side-Navbar 
     $(document).ready(function(){
+    $('li').click(function(){
+        $('li').removeClass("active");
+        $(this).addClass("active");
+        });
+    });
+
+    //Show Info on hover over Show Active Courses
+    $(document).ready(function(){
+     $('[data-toggle="tooltip"]').tooltip(); 
```

Comment:
I think the link name should be self-explanatory, this is completely unnecessary